src/Aspire.Hosting/build/Aspire.Hosting.targets

```diff
@@ -114,11 +114,13 @@ internal static class ]]>%(ClassName)<![CDATA[
   <Target Name="WriteServiceMetadataSources" DependsOnTargets="$(_WriteServiceMetadataSourcesDependsOn)" BeforeTargets="CoreCompile" />
 
   <!-- This target registers the location of the Aspire orchestration dependencies -->
-  <Target Name="SetOrchestrationDiscoveryAttributes" BeforeTargets="GetAssemblyAttributes">
+  <Target Name="SetOrchestrationDiscoveryAttributes" BeforeTargets="GetAssemblyAttributes" Condition=" '$(DcpDir)' != '' ">
```

Comment:
Can something potentially break from making this whole target conditional where it wasn't before? (Before we were defaulting DcpDir if not set)

Not super familiar with these targets, so asking just in case.

Reply:
It'll change the error the user sees; currently we report that we can't find the orchestrator at the fallback DcpDir path we were setting, but that path was a holdover from the pre-workload days. After the change, we'll throw a different error in DcpOptions.ApplyApplicationConfiguration when we see the CliPath wasn't set by the expected attribute.